src/ngraph/op/topk.hpp

```diff
@@ -196,34 +191,60 @@ namespace ngraph
                 virtual void generate_adjoints(autodiff::Adjoints& adjoints,
                                                const OutputVector& deltas) override;
 
-                size_t read_k_from_constant_node(const std::shared_ptr<Node>& node,
-                                                 const element::Type& k_element_type) const;
-
-                Mode mode_from_string(const std::string& mode) const;
-                SortType sort_type_from_string(const std::string& sort) const;
+                virtual size_t read_k_from_constant_node(const std::shared_ptr<Node>& node,
+                                                         const element::Type& k_element_type) const;
 
                 template <typename T>
                 size_t validate_and_get_k(const std::shared_ptr<op::Constant>& k_constant) const;
             };
         } // namespace v1
 
-        using v0::TopK;
-    } // op
+        namespace v3
```

Comment:
In "Adding a new Op" instruction there is: "If the opset3 op is a new version of an existing op, put the new definition earlier in the file, so that you see the current version of the op first". In this case `TopK::v1` foward declaration will be needed